lib/auth/tls_test.go

```diff
@@ -1807,6 +1807,107 @@ func TestExtendWebSessionWithReloadUser(t *testing.T) {
 	require.Equal(t, traits[constants.TraitDBUsers], []string{"llama", "alpaca"})
 }
 
+func TestExtendWebSessionWithMaxDuration(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	testSrv := newTestTLSServer(t)
+	clock := testSrv.AuthServer.TestAuthServerConfig.Clock
+
+	clt, err := testSrv.NewClient(TestAdmin())
```

Comment:
```suggestion
	adminClient, err := testSrv.NewClient(TestAdmin())
```

There are too many "clients" and too many vars to keep track here, so I'm suggesting a few "bigger" names. Feel free to tweak accordingly, I'm not sure I got them all right.